### PR TITLE
tlg0059.tlg037

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated URN and metadata

### CODE REVIEW

File: data/tlg0059/tlg037/tlg0059.tlg037.1st1K-grc1.xml [hunk 1]
```diff
@@ -4,7 +4,8 @@
 <teiHeader xml:lang="eng">    
 <fileDesc>
 <titleStmt>
-<title xml:lang="lat">Definitiones</title>
+ <title xml:lang="grc">Ὅροι</title>
+ <author>Plato</author>
```
Comment (lcerrato) [inline data/tlg0059/tlg037/tlg0059.tlg037.1st1K-grc1.xml:8]: you have two authors

File: data/tlg0059/tlg037/tlg0059.tlg037.1st1K-grc1.xml [hunk 2]
```diff
@@ -50,25 +51,21 @@
 <pubPlace>United States</pubPlace>
 </publicationStmt>
 <sourceDesc>
-<listBibl xml:lang="lat">
+<listBibl>
```
Comment (lcerrato) [inline data/tlg0059/tlg037/tlg0059.tlg037.1st1K-grc1.xml:54]: `<listBibl>` is not standard inclusion?

**Review by lcerrato — APPROVED** [review]
made changes

File: data/tlg0059/tlg037/tlg0059.tlg037.1st1K-grc1.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/tlg0059/tlg037/tlg0059.tlg037.1st1K-grc1.xml]: hi @lcerrato you are right, I think I had just gotten out of habit of eliminating it. Thanks!

**Pull request merged**